frontend/zc_messaging/src/utils/fetchDefaultRoom.js

```diff
@@ -0,0 +1,16 @@
+import axios from "axios"
+
+const fetchDefaultRoom = async (orgId, userId) => {
+  const response = await axios.get(
+    `https://chat.zuri.chat/api/v1/sidebar?org=${orgId}&user=${userId}`
+  )
+  const { public_rooms } =
+    response.data?.data?.find(room => room.name === "Channels") || {}
+  const defaultRoom =
+    public_rooms?.find(room => room.room_name === "general") || {}
+  return {
+    room_id: defaultRoom.room_id
```

Comment:
The case for defaultRoom if its null, doesn't have a room_id-> {}

Reply:
won't throw an error, will just be undefined